compiler/natives/src/internal/reflectlite/reflectlite.go

```diff
@@ -0,0 +1,1048 @@
+// +build js
```

Comment:
So far I tool only cursory look at `reflectlite` code. I assume it is mostly derived from GopherJS version of `reflect`?

Is it possible to somehow reduce the amount of code duplication between the two?

Reply:
reflectlite clone from goperhjs reflect and comment unused code.